### DISCUSSION TITLE
Zigbee2mqtt multiple instances - cluster

### DISCUSSION THREAD

**Discussion by begemotik** [Koenkk/zigbee2mqtt#21112] [General]
I've tried to search through the discussion and found few topics, but the case there is quite different from what I'd like to reach.
I'm wondering if it's possible to have multiple zigbee2mqtt coordinators within the same network, so that if one dongle becomes unavailable for some reason all the devices could be still managed by other instance.

Is it possible?

**Comment by Koenkk** [comment]
No, a Zigbee network can have only 1 coordinator.
